zboxcore/sdk/filerefsworker.go

```diff
@@ -61,7 +62,11 @@ func (o *ObjectTreeRequest) GetRefs() (*ObjectTreeResult, error) {
 	hashCount := make(map[string]int)
 	hashRefsMap := make(map[string]*ObjectTreeResult)
 
+	found := false
 	for _, oTreeResponse := range oTreeResponses {
+		if oTreeResponse.statusCode != http.StatusBadRequest { //It should be http.StatusNotFound, but the blobber is returning 400, if the file not found
```

Comment:
Collect errors and return the true major error. Check how it processed the errors in [ProcessCopy](https://github.com/0chain/gosdk/blob/a751ddada893441aa6b56a2b5baa060dc9a8409b/zboxcore/sdk/copyworker.go#L174)